app/templates/my-sessions/view.hbs

```diff
@@ -8,9 +8,11 @@
     {{#if device.isMobile}}
       <div class="ui hidden fitted divider"></div>
     {{/if}}
-    <button class="ui red button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Withdraw Proposal'}}</button>
-    {{#if device.isMobile}}
-      <div class="ui hidden fitted divider"></div>
+    {{#if ifpast}}
```

Comment:
This is counter intuitive, if an event is `isPast`,  i.e. in the past, you want to hide the button. Use 
```hbs
{{#unless isPast}}
<button class="ui red button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Withdraw Proposal'}}</button>
     {{#if device.isMobile}}
        <div class="ui hidden fitted divider"></div>
     {{/if}}
{{/unless}}

Reply:
No, the condition will be if only. I'll change the variable name to isUpcoming. Thanks :smile: